tests/haskell.vader

```diff
@@ -0,0 +1,39 @@
+Include: _setup.vader
```

Comment:
Oh, right. I tried to put it under `tests/ft` initially but I couldn't get it to work on the first try. I wouldn't mind if you do it. :grinning: